data/templates/get_driveid.js

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env node
+
+'use strict';
+
+var exec = require('child_process').exec;
+
+var cmdDriveWwid = 'ls -l /dev/disk/by-id';
+var cmdVdInfo = 'ls -l /dev/disk/by-path';
```

Comment:
I think we should change this variable name, this command is not to fetch the virtual drive information